app/server/lib/Comm.js

```diff
@@ -150,6 +151,19 @@ Comm.prototype._broadcastMessage = function(type, data, clients) {
   clients.forEach(client => client.sendMessage({type, data}));
 };
 
+
+/**
+ * Returns a profile based on the request or session.
+ */
+Comm.prototype._getSessionProfile = function(scopedSession, req) {
+  const profile = getRequestProfile(req);
+  if (profile)
```

Comment:
Small lint: local coding style is to never omit curly-parentheses.

Reply:
parentheses added